__tests__/app.test.js

```diff
@@ -149,6 +149,37 @@ describe('/api/reviews/:review_id/comments', () => {
     });
 })
 
+describe('/api/reviews/:review_id/comments', () => {
+    test('POST- status:201- responds with the added comment object', () => {
+        const newComment = { username: 'bainesface', body: 'this is the story about a pickle' }
+        return request(app).post('/api/reviews/1/comments').send(newComment).expect(201).then(({ body }) => {
+            expect(body.newComment).toBeInstanceOf(Object)
+            expect(body.newComment.author).toBe('bainesface')
+            expect(body.newComment.body).toBe('this is the story about a pickle')
+        })
+    })
+
```

Comment:
Please add a 201 test for ignoring unnecessary properties on the comment object. Also add 400 test for an invalid id